### ISSUE TITLE
After going through "forgot password" ended up with two login credentials

### ISSUE THREAD

**Issue by skofman1** [NuGet/NuGetGallery#6716]
Repro steps:
1. I'm an admin on dev environment
2. Logged in with AAD
3. Logged out
4. Used "forget passowrd" to reset password

Result:
1. Got a page asking to "link my account"
2. After linking my account has two logins (same e-mail):
![image](https://user-images.githubusercontent.com/16807822/49303506-ac629a80-f47e-11e8-864e-92484d06b830.png)

**Comment by skofman1** [comment]
@shishirx34 , what do you think?

**Comment by scottbommarito** [comment]
I think one account is an `AzureActiveDirectoryV1` login and the other is an `AzureActiveDirectoryV2` login. Previously, all admins had both, but then we cleaned it up.

Are you sure you didn't have both before? If you're sure the second login is new, this is probably a bug with us still allowing `AzureActiveDirectoryV1` login. If it was existing, it probably wasn't cleaned up before.

**Comment by shishirx34** [comment]
Yeah, its the V1 login only happens for admins. we'll have to fix your account.
